js/config.ts

```diff
@@ -180,4 +187,10 @@ Object.keys(DEFAULT_CONFIG).forEach((key) => {
 });
 
 const finalConfig: Record<string, any> = { ...DEFAULT_CONFIG, ...urlConfig };
+// `geode` param overrides some other params
+if (finalConfig.geode) {
+  // can't draw force arrows if you can't rotate the globe
```

Comment:
The advantage of fixing it at runtime like this is that you can just add `geode` to a TecRocks url and get reasonable functionality. Otherwise, to get a functioning geode application you have to specify `geode&cameraLockedInPlanetWizard=false` to get a usable geode instance. The request in the story is that assigning forces in geode should behave the old way, not that it should be configurable to sometimes work the old way and sometimes in a hybrid way that doesn't seem to make sense. If we don't do _something_ about this then at some point down the road someone will go to the authoring page, click the `geode` checkbox, and then file a bug that the planet wizard is broken in geode because you can't rotate the globe. It seems to me that if we don't put some effort into reducing the error-prone-ness of the authoring page and url params then we just create potential for confusion of the type that seems to recur with LARA authoring.

Reply:
Right. In this particular case, I could also imagine `cameraLockedInPlanetWizard=true` being ignored when `geode=true`. But I see your point.